src/assets/styles/_overrides.scss

```diff
@@ -0,0 +1,5 @@
+// Global overrides
+
+.btn {
+  border-radius: 0;
+}
```

Comment:
What I would suggest is to create an overrides folder with an index.scss and then create files for specific overrides.

Also, for getting this into master, I already commented using the global variable that Boostrap 4 provides.

Reply:
Oh, that's great. Didn't see the global flag, I assume it would take care of all rounded corners (input fields, modals, etc)? So I'll just set that variable to false, then remove this overrides and we can worry about downstream specific overrides later as we need?